default/scripting/techs/ship_hulls/energy/SHP_FRC_ENRG_COMP.focs.py

```diff
@@ -12,6 +12,7 @@
     unlock=[
         Item(type=UnlockShipHull, name="SH_COMPRESSED_ENERGY"),
         Item(type=UnlockBuilding, name="BLD_SHIPYARD_ENRG_COMP"),
+        Item(type=UnlockPolicy, name="PLC_CHARGE"),
```

Comment:
? not in log message or summary...

Reply:
Added it to commit message. Or did you want me to remove the unlock?